appsrc/styles.less

```diff
@@ -0,0 +1 @@
+@import "../node_modules/bootstrap/less/bootstrap.less";
```

Comment:
I don't know much about less, but I'm assuming we don't need an empty line at the end of the file?